File: src/kyu_8/the_wide_mouthed_frog/csharp/the_wide_mouthed_frog_test.cs
namespace Solution {
  using NUnit.Framework;
  using System;

  [TestFixture]
  public class SolutionTest
  {
    [Test]
    public void SampleTest()
    {
      Assert.AreEqual("wide", Kata.MouthSize("toucan"));
      Assert.AreEqual("wide", Kata.MouthSize("ant bear"));
      Assert.AreEqual("small", Kata.MouthSize("alligator"));
    }
  }
}

File: src/kyu_8/the_wide_mouthed_frog/python/the_wide_mouthed_frog_test.py
import codewars_test as test
from solution import mouth_size

@test.describe("Fixed Tests")
def fixed_tests():
    @test.it('Basic Test Cases')
    def basic_test_cases():
        test.assert_equals(mouth_size("toucan"),"wide")
        test.assert_equals(mouth_size("ant bear"),"wide")
        test.assert_equals(mouth_size("alligator"),"small")
